configure

```diff
@@ -27,7 +27,7 @@ CONFIGURE_ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 if [[ "$PYTHON_EXE" == "" ]]; then
     PYTHON_EXE=python
 fi
-
+PYTHON_EXE=python2.7
```

Comment:
What is this for?

Reply:
Actually I had 3 versions of python in my system to explicitly configure with py2.7 I was using it,else it was showing config errors(I am going to change it  now)